controllers/apply-next/views/dashboard.njk

```diff
@@ -1,157 +1,32 @@
-{% from "components/buttons.njk" import startButton, startButtonSubmit %}
+{% from "components/application-card/macro.njk" import applicationCard with context %}
 {% from "components/content-box/macro.njk" import contentBox %}
-{% from "components/form-header/macro.njk" import formHeader with context %}
-{% from "components/form-fields/macros.njk" import formErrors, formField with context %}
-{% from "components/icons.njk" import iconBin %}
-{% from "./view-helpers.njk" import showSectionProgress with context %}
 
 {% extends "layouts/main.njk" %}
 
-{% set fieldMissing %}<span class="application-data-missing">not yet completed</span>{% endset %}
```

Comment:
Any reason for removing this? If it's no longer in the design then the CSS can go, too.

Reply:
I think this is me being overeager, this should stay.